### ISSUE TITLE
Missing more... for updates and tutorials in /gaming

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#792]
```
No apparent way to view more... updates and more... tutorials in /gaming
```

Original issue reported on code.google.com by `sethladd@google.com` on 1 Feb 2012 at 2:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:13
- Changed state: **Assigned**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c15622f2654d.
```

Original comment by `ericbide...@html5rocks.com` on 3 Feb 2012 at 11:31
- Changed state: **Fixed**
